### PR TITLE
Gets Rid of "Banned Content" and references of /tg/ in Contributing.md

### PR DESCRIPTION
**Pull request by Sarcalogo**
See title.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and fo
 
 ##Introduction
 
-Hello and welcome to /tg/station's contributing page. You are here because you are curious or interested in contributing. Thanks for being interested. Everyone is free to contribute to this project as long as they follow the simple guidelines and specifications below, because at /tg/station, we have a goal to increase code maintainability and to do that we are going to need all pull requests to hold up to those specifications. This is in order for all of us to benefit, instead of having to fix the same bug more than once because of duplicated code.
+Hello and welcome to Hippiestation's contributing page. You are here because you are curious or interested in contributing. Thanks for being interested. Everyone is free to contribute to this project as long as they follow the simple guidelines and specifications below, because at Hippiestation, we have a goal to increase code maintainability and to do that we are going to need all pull requests to hold up to those specifications. This is in order for all of us to benefit, instead of having to fix the same bug more than once because of duplicated code.
```
Comment (JamieH) [inline CONTRIBUTING.md]: Hippiestation should be HippieStation

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,16 +6,16 @@ See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and fo
...
 
 If you want to contribute the first thing you'll need to do is [set up Git](http://tgstation13.org/wiki/Setting_up_git) so you can download the source code.
 
-We have a [list of guides on the wiki](http://www.tgstation13.org/wiki/index.php/Guides#Development_and_Contribution_Guides) which will help you get started contributing to /tg/station with git and Dream Maker. For beginners, it is recommended you work on small projects, at first. If you need help learning to program in BYOND check out this [repository of resources](http://www.byond.com/developer/articles/resources).
+We have a [list of guides on the wiki](http://www.tgstation13.org/wiki/index.php/Guides#Development_and_Contribution_Guides) which will help you get started contributing to Hippiestation with git and Dream Maker. For beginners, it is recommended you work on small projects, at first. If you need help learning to program in BYOND check out this [repository of resources](http://www.byond.com/developer/articles/resources).
```
Comment (JamieH) [inline CONTRIBUTING.md]: Correct the links to our wiki etc.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,16 +6,16 @@ See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and fo
...
 
 There is an open list of approachable issues for [your inspiration here](https://github.com/tgstation/-tg-station/issues?q=is%3Aopen+is%3Aissue+label%3A%22Easy+Fix%22).
```
Comment (JamieH) [inline CONTRIBUTING.md]: Remove this too

**Comment by Sarcalogo** [comment]
Need the official URL for coder chat and then I can add that and we're done here.

**Comment by Sarcalogo** [comment]
Ah woops I messed something up, going to have to close indefinitely.

**Pull request closed without merging**